process/errors.go

```diff
@@ -569,6 +569,9 @@ var ErrNilEconomicsData = errors.New("nil economics data")
 // ErrZeroMaxComputableRounds signals that a value of zero was provided on the maxComputableRounds
 var ErrZeroMaxComputableRounds = errors.New("max computable rounds is zero")
 
+// ErrZeroMaxConsecutiveRoundsOfRatingDecrease signals that a value of zero was provided on the MaxConsecutiveRoundsOfRatingDecrease
+var ErrZeroMaxConsecutiveRoundsOfRatingDecrease = errors.New("max consecutive number of rounds in which we can decrease a validator rating is zero")
```

Comment:
max consecutive number of rounds**,** in which we can decrease a validator rating**,** is zero

Reply:
Added **,**

Reply:
ok!